.dockerignore

```diff
@@ -4,4 +4,3 @@
 .gitattributes
 READMETEMPLATE.md
 README.md
```

Comment:
The target dir for the docker ignore should probably still be there for local testing of the image.

Reply:
I'm copying the binaries out of the target directory... I can probably put a step in the build process to consolidate the binaries elsewhere?

Reply:
Hmm. You make a good point. This may break local testing. We may need a Dockerfile.local or another workaround.

Reply:
It definitely will break local testing, but your idea about a `Dockerfile.local` should fix it.